openapi-cli/src/test/java/io/ballerina/openapi/CodeGeneratorTest.java

```diff
@@ -90,6 +91,7 @@ public void generateClient() {
         CodeGenerator generator = new CodeGenerator();
         try {
             String expectedClientContent = getStringFromGivenBalFile(expectedServiceFile, "generate_client.bal");
+            generator.setIncludeTestFiles(true);
```

Comment:
Shall we add separate tests for `--with-test` flag for service generation, client generation, both service and client generation scenarios instead of changing already existing tests 🙂 it will be more clear for filter out the test with flag in command WDYT?

Reply:
+1. Adding a separate test case for --with-test scenario will be more clear.

Reply:
There is a separate test case for --with-test but here what we have done is that enabling the addition of tests/test.bal and generation other test-related files as the function was designed earlier, without changing the expected output of the test cases.